### PR TITLE
net-misc/dleyna-server: fix bug #597952

### PR DESCRIPTION
**Pull request by kobboi**
Package-Manager: portage-2.3.2

### CODE REVIEW

File: net-misc/dleyna-server/dleyna-server-0.5.0.ebuild [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```
Comment (SoapGentoo) [inline net-misc/dleyna-server/dleyna-server-0.5.0.ebuild]: this is a stable ebuild. Leave it alone, and revbump it rather (to `~amd64`, using `ebump -C`).

File: net-misc/dleyna-server/dleyna-server-0.5.0.ebuild [hunk 2]
```diff
@@ -29,6 +29,9 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig
 "
 
+# Bug 597952
+PATCHES=( "${FILESDIR}"/${P}-fix-references-to-GUPnPContextManager.patch )
+
 src_install() {
 	default
 	prune_libtool_files
```
Comment (SoapGentoo) [inline net-misc/dleyna-server/dleyna-server-0.5.0.ebuild]: Check that the `.la` files can be deleted and then replace this with the find command from https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives . Then you can also nix the `inherit eutils`

File: net-misc/dleyna-server/files/dleyna-server-0.5.0-fix-references-to-GUPnPContextManager.patch [hunk 3]
```diff
@@ -0,0 +1,10 @@
+-- -a/libdleyna/server/upnp.h
```
Comment (SoapGentoo) [inline net-misc/dleyna-server/files/dleyna-server-0.5.0-fix-references-to-GUPnPContextManager.patch]: `--- a`, not `-- -a` :stuck_out_tongue:

File: net-misc/dleyna-server/dleyna-server-0.5.0.ebuild (hunk 1 above)
Comment (kobboi) [inline net-misc/dleyna-server/dleyna-server-0.5.0.ebuild]: Keeping EAPI=5 and adding a patch as agreed on IRC

File: net-misc/dleyna-server/files/dleyna-server-0.5.0-fix-references-to-GUPnPContextManager.patch (hunk 3 above)
Comment (kobboi) [inline net-misc/dleyna-server/files/dleyna-server-0.5.0-fix-references-to-GUPnPContextManager.patch]: Busted! Fixed.

File: net-misc/dleyna-server/dleyna-server-0.5.0.ebuild (hunk 2 above)
Comment (kobboi) [inline net-misc/dleyna-server/dleyna-server-0.5.0.ebuild]: Package does not install static libraries, nor does it have a USE flag that would make it so. I replaced the {{prune_libtool_files}} and {{inherit eclass}}

**Comment by EvaSDK** [comment]
Reverted half-EAPI up/downgrade and pushed. Thanks.

**Pull request closed without merging**